packages/apps/plugins/plugin-sketch/src/SketchPlugin.tsx

```diff
@@ -166,6 +167,7 @@ export const SketchPlugin = (): PluginDefinition<SketchPluginProvides> => {
               // NOTE: Min 500px height (for tools palette).
               return data.object instanceof DiagramType ? (
                 <SketchComponent
+                  key={fullyQualifiedId(data.object)} // Force instance per sketch object. Otherwise, sketch shares the same instance.
```

Comment:
is fully qualified id enough? there's no issue if the same sketch is on the screen twice?

Reply:
Yeah it's working. Keyboard actions are still shared between instances of the *same* sketch, but it's kinda cool.

Reply:
The background stuff was far more jarring.